community/modules/scheduler/schedmd-slurm-gcp-v5-controller/README.md

```diff
@@ -67,6 +67,18 @@ activated through the `enable_reconfigure` setting:
     enable_reconfigure: true
 ```
 
+To reconfigure a running cluster:
+
+1. Edit the blueprint with the desired configuration changes
+1. Call `ghpc create <blueprint> -w` to overwrite the deployment directory
+1. Follow instructions in terminal to deploy
+
+The following are examples of updates that can be made to a running cluster:
+
+* Add or remove a partition to the cluster
+* Resize an existing partition
+* Attach new network storage to an existing partition
+
```

Comment:
Optional: maybe add that  changing the VM type of a partition will likely not work. (I tried with a gpu vm today and it did not work -- as it fails to update the instance template). It seems like a common case.

Reply:
Done.